tests/nlu/featurizers/test_lm_featurizer.py

```diff
@@ -366,10 +391,33 @@ def test_attention_mask(
     assert np.all(mask_zeros == 0)
 
 
-# TODO: need to fix this failing test
-@pytest.mark.skip(reason="Results in random crashing of github action workers")
+def skip_on_CI(model_name, model_weights):
+    # This only applies when skip_model_load=False
+    return model_name == "bert" and bool(os.environ.get("CI")) and not model_weights
+
+
+def create_config(model_name, model_weights):
```

Comment:
`create_config` is a bit general. Could we rename this to something like `create_config_for_lm_featurizer`?

Reply:
Also return types and argument types

Reply:
You're right, it is vague! I'm not sure about tying it to `lm_featurizer` since it's also used (for now) by `HFTransformersNLP`. Do you thunk that's confusing or maybe not such a big deal? What do you think of `create_transformers_config`?

Reply:
Yes, you're right it is vague! I'm not sure about tying it to `LMFeaturizer`, I wonder if that'd be confusing for the time it's used in `HFTransformersNLP`? Or maybe it's not a big deal? I think maybe `create_pretrained_transformers_config` could work. It's long but since it's just a test helper function..